src/fhir-tool-core/Operations/GenerateBinaryOperation.cs

```diff
@@ -0,0 +1,89 @@
+﻿extern alias R4;
+using CommandLine;
+using FhirTool.Core.ArgumentHelpers;
+using R4::Hl7.Fhir.Model;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+using FhirTool.Core.FhirWrappers;
+using System.IO;
+using System.Linq;
+
+namespace FhirTool.Core.Operations
+{
+    [Verb("generate-binary", HelpText = "Generates binary file")]
+    public class GenerateBinaryOperationOptions
+    {       
+        [Option('i', "id", HelpText = "Binary id", Required = true)]
+        public string Id { get; set; }
+
+        [Option('c', "contentType", HelpText = "Content type", Required = true)]
```

Comment:
For konformitet med argumenter i andre operasjoner bruk bindestrek '-' for å binde sammen flere ord i stedet for camelCase: `[Option('c', "content-type", HelpText = "Content-Type for attachment", Required = true)]`